docs/source/index.rst

```diff
@@ -1,22 +1,33 @@
-.. JupyterHub documentation master file, created by
-   sphinx-quickstart on Mon Jan  4 16:31:09 2016.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 JupyterHub
 ==========
 
-.. note:: This is the official documentation for JupyterHub. This project is
-          under active development.
```

Comment:
I removed this note because both of the things it said seemed like the default assumptions, and we'd only need to mention it if they weren't the case. Also, I don't think it's useful to claim in docs that it's actively developed, because from the reader's point of view, there's no guarantee that that's still true.